tests/components/moehlenhoff_alpha2/test_config_flow.py

```diff
@@ -0,0 +1,83 @@
+"""Test the moehlenhoff_alpha2 config flow."""
+import asyncio
+from unittest.mock import PropertyMock, patch
+
+from moehlenhoff_alpha2 import Alpha2Base
+
+from homeassistant.components.moehlenhoff_alpha2.const import DOMAIN
+from homeassistant.helpers.typing import HomeAssistantType
```

Comment:
Change `HomeAssistantType` to `HomeAssistant`:
https://github.com/home-assistant/core/blob/c74f9a8313c621c9d8c777b07216c286d55d6118/homeassistant/helpers/typing.py#L28-L33

Reply:
Done.